homeassistant/components/backup/websocket.py

```diff
@@ -260,9 +260,12 @@ async def backup_agents_download(
         )
         return
     try:
-        path = manager.temp_backup_dir / f"{msg["backup_id"]}.tar"
+        # Note: We should remove this WS command, it's only used in tests,
+        # tests can instead call GET /api/backup/download/{backup_id}
+        temp_backup_dir = manager._reader_writer.temp_backup_dir  # noqa: SLF001
```

Comment:
A follow-up PR will remove the unused WS command `backup/agents/download` and adjust affected tests to call HTTP GET `/api/backup/download/{backup_id}` instead.